pubsub/driver/driver.go

```diff
@@ -38,6 +38,24 @@ type Batcher interface {
 // AckID is the identifier of a message for purposes of acknowledgement.
 type AckID interface{}
 
+// AckAction is an enum of possible actions on an AckID.
+type AckAction int
+
+const (
```

Comment:
```
const (
  //  ... 
   ActionAck AckAction = iota
  //
   ActionNack
)
```
or `iota+1` if you really want to start from 1 (would require a justifying comment).

Reply:
BTW, this is fine if you want, but a simple bool would do too.

Reply:
Done. You are right, bool is simpler. I used an enum because I was thinking of us adding "extend ack deadline" later, but that would probably require another value anyway, so let's worry about that later.